tests/test_mpi.py

```diff
@@ -850,19 +850,17 @@ def test_hoist_haloupdate_with_subdims(self):
 
         op = Operator(eqns)
 
-        assert len(op._func_table) == 6
+        assert len(op._func_table) == 4
 
         # There are exactly two halo exchange calls in the Operator body
         calls = FindNodes(Call).visit(op)
-        assert len(calls) == 2 + 8  # 8 are due to loop blocking
+        assert len(calls) == 2 + 0  # 0 are due to loop blocking
```

Comment:
nitpicking: you may now drop the + 0 part

Reply:
dropped